--- /usr/share/phpmyadmin/libraries/Config.class.php.orig	2017-01-30 06:30:24.303943410 +0000
+++ /usr/share/phpmyadmin/libraries/Config.class.php	2017-01-30 06:34:41.487790064 +0000
@@ -1294,13 +1294,6 @@
             }
             // Add hostname
             $pma_absolute_uri .= $url['host'];
-            // Add port, if it not the default one
-            if (! empty($url['port'])
-                && (($url['scheme'] == 'http' && $url['port'] != 80)
-                || ($url['scheme'] == 'https' && $url['port'] != 443))
-            ) {
-                $pma_absolute_uri .= ':' . $url['port'];
-            }
             // And finally path, without script name, the 'a' is there not to
             // strip our directory, when path is only /pmadir/ without filename.
             // Backslashes returned by Windows have to be changed.
